### PR TITLE
Added section about cache_path

### PR DESCRIPTION
**Pull request by bphermansson**
See #260

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,12 @@ the `[global]` section. This is useful when you run applications related to
 Values can be surrounded by double quotes (") which is useful if the value contains
 the comment character (#).
 
+The line "cache_path = cache_directory" defines where Spotify's cache-files are stored. 
+These files are used to avoid downloading a file again when a track is replayed. 
+Here it's set to "cache_directory" and the cache files will be stored in "/cache_directory".
```
Comment (clushie) [inline README.md]: I think
Here it's => Here its

https://grammarist.com/spelling/its-its/

Comment (clushie) [reply README.md]: > and the cache files will be stored in "/cache_directory".

Might be worth mentioning that:
 - tilde-expansion `~/cache/spotifyd` won't work
 - variable `${HOME}/cache/spotifyd` won't work
 - the directory needs to exist

File: README.md [hunk 2]
```diff
@@ -70,6 +70,12 @@ the `[global]` section. This is useful when you run applications related to
 Values can be surrounded by double quotes (") which is useful if the value contains
 the comment character (#).
 
+The line "cache_path = cache_directory" defines where Spotify's cache-files are stored. 
+These files are used to avoid downloading a file again when a track is replayed.
```
Comment (SirWindfield) [inline README.md]: These files are used to avoid re-downloading a file if a track is replayed.

**Comment by SirWindfield** [comment]
Thanks for helping us out!

File: README.md (hunk 1 above)
Comment (SimonPersson) [reply README.md]: > I think
> Here it's => Here its
> 
> https://grammarist.com/spelling/its-its/

No, it is good as it is.

**Comment by SimonPersson** [comment]
Tack @bphermansson!

@SirWindfield and @clushie - I pushed a commit that addresses your comments. Thanks!

**Comment by bphermansson** [comment]
No problem, thanks for a great software!

**Comment by clushie** [comment]
Thank you too!

**Pull request merged**